### PR TITLE
#731 - Resolve puzzler

### PR DESCRIPTION
**Pull request by neonailol**

### CODE REVIEW

**Comment by 0crat** [comment]
[@yegor256](http://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#768](https://github.com/yegor256/takes/issues/768) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#768](https://github.com/yegor256/takes/issues/768) is assigned to [@g4s8](http://www.0crat.com/u/g4s8). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and [§10](http://datum.zerocracy.com/pages/policy.html#10) and when you decide to accept the changes, inform [@yegor256](http://www.0crat.com/u/yegor256) right in this ticket.

File: src/test/java/org/takes/tk/TkSlf4jTest.java [hunk 1]
```diff
@@ -44,9 +44,6 @@
  * @todo #712:30min Prepare implementation for empty response body test and
  *  unignore returnsAnEmptyResponseBody test to fix such error which was
  *  reported in #712.
- * @todo #712:30min Refactor this class to reduce the data abstraction coupling
- *  in order to get rid of the checkstyle suppression of
- *  ClassDataAbstractionCouplingCheck.
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```
Comment (g4s8) [inline src/test/java/org/takes/tk/TkSlf4jTest.java]: @neonailol it's `ClassDataAbstractionCouplingCheck` checkstyle suppression so far, looks like this puzzle was not resolved

**Comment by neonailol** [comment]
@g4s8 removed the todo, but forgot the checkstyle comment, pr updated

**Comment by g4s8** [comment]
@neonailol try to build it with Qulice plugin (`mvn clean install -Pqulice --errors`), you will get an error, as in [this build](https://ci.appveyor.com/project/yegor256/takes/build/2902):
> [INFO] Checkstyle: C:\projects\takes\src\test\java\org\takes\tk\TkSlf4jTest.java[48]: Class Data Abstraction Coupling is 10 (max allowed is 7) classes [FtRemote, FtRemote.Script, IOException, IsEqual, JdkRequest, RqFake, TkEmpty, TkFailure, TkSlf4j, TkText]. (ClassDataAbstractionCouplingCheck)

please, see [checkstyle docs](http://checkstyle.sourceforge.net/config_metrics.html):
> This metric measures the number of instantiations of other classes within the given class. This type of coupling is not caused by inheritance or the object oriented paradigm. Generally speaking, any data type with other data types as members or local variable that is an instantiation (object) of another class has data abstraction coupling (DAC). The higher the DAC, the more complex the structure of the class.

**Comment by g4s8** [comment]
@yegor256 this puzzle seems to be resolved

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/takes#768` is not assigned, can't get start time

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @g4s8[/z](http://www.0crat.com/u/g4s8), total is [+3190](http://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](http://www.0crat.com/u/yegor256), total is [+7580](http://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#768](https://github.com/yegor256/takes/issues/768) is now out of scope

**Comment by 0crat** [comment]
Job [#768](https://github.com/yegor256/takes/issues/768) was not in scope

**Pull request merged**